.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - checkout
       - node/install:
-          node-version: '16.14'
+          node-version: '20.16'
```

Comment:
This is only for the test environment.

Reply:
I know only for testing but also hooray for parity with `vxsuite`!